packages/mobile/src/app/saga.ts

```diff
@@ -184,6 +197,9 @@ export function* appRemoteFeatureFlagSaga() {
       if (configValues) {
         yield put(updateRemoteConfigValues(configValues))
         lastLoadTime = Date.now()
+        if (configValues.allowOtaTranslations) {
+          yield put(fetchOtaTranslations())
```

Comment:
Though this works, we could decouple this and have a watcher for the `UPDATE_REMOTE_CONFIG_VALUES` action type. The watcher could also listen to the `SET_LANGUAGE` action type to trigger the OTA fetch.
And the explicit `fetchOtaTranslations` wouldn't be needed.

Though the current codebase is not a good example of this, it's preferable to avoid dispatching multiple actions sequentially.
See https://redux.js.org/style-guide/style-guide#avoid-dispatching-many-actions-sequentially and https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters

Reply:
yeah good call out, i've removed the `fetchOtaTranslations` action and it's simplified the logic a lot i think :)